packages/bsky/src/api/com/atproto/moderation/createReport.ts

```diff
@@ -1,42 +0,0 @@
-import { AuthRequiredError } from '@atproto/xrpc-server'
```

Comment:
If it's useful to the rollout, we could pass requests to here through to automod for a short time period.

Reply:
hmmm yup yup good point 👍